tests/validation/tests/v3_api/test_v2_cis_scan.py

```diff
@@ -0,0 +1,118 @@
+import pytest
+import time
+import os
+from .common import USER_TOKEN
+from .common import get_cluster_client_for_token_v1
+from .common import execute_kubectl_cmd
+from .common import get_user_client_and_cluster
+from .common import DEFAULT_TIMEOUT
+CIS_CHART_VERSION = os.environ.get('RANCHER_CIS_CHART_VERSION', "1.0.100")
+SCAN_PROFILE = os.environ.get('RANCHER_SCAN_PROFILE', "rke-profile-permissive")
+
+cluster_detail = {"cluster": None}
+cis_annotations = \
+    {
+        "catalog.cattle.io/ui-source-repo": "rancher-charts",
+        "catalog.cattle.io/ui-source-repo-type": "cluster"
+    }
+cis_charts = {
+    "values":
+        { "global": {"cattle":{"clusterId": None, "clusterName": None}}},
+    "version": CIS_CHART_VERSION,
+    "projectId": None
+}
+
+
+def test_install_v2_cis_benchmark():
+    client = \
+        get_cluster_client_for_token_v1(
+            cluster_detail["cluster"]["id"],USER_TOKEN
+        )
+    rancherrepo = \
+        client.list_catalog_cattle_io_clusterrepo(id="rancher-charts")
+    print(rancherrepo["data"][0])
+    cluster_id = cluster_detail["cluster"]["id"]
+    cluster_name = cluster_detail["cluster"]["name"]
+    rancher_repo = rancherrepo["data"][0]
+
+    # check if CIS is already installed
+    app = client.list_catalog_cattle_io_app()
+    print("app", app)
+    for app in app["data"]:
+        print("app=",app)
+        if app["metadata"]["name"] == "rancher-cis-benchmark":
+            response = client.action(obj=app, action_name="uninstall")
+        if app["metadata"]["name"] == "rancher-cis-benchmark-crd":
+            response = client.action(obj=app, action_name="uninstall")
+    time.sleep(10)
```

Comment:
Did you test the case where the app is installed? Is 10s sleep long enough?
Can you see if we can use the existing method `wait_for_app_to_remove`, or we need to make a v1 version of it?

Reply:
Yes i tested where app is installed, and I had to delete it first and then it installed it back. But yes let me modify this and put a wait method. I think we have to create a new one for the v2 apps.

Reply:
Added `wait_until_app_v2_uninstall`